addons/mass_mailing_crm/tests/test_mass_mailing_crm.py

```diff
@@ -0,0 +1,28 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.tests.common import Form, TransactionCase
+
+
+class TestMassMailingCrm(TransactionCase):
```

Comment:
Would use ``MassMailCase`` to benefits from all tooling and preparation done there :)